catalog/MDCCatalog/MDCMenuViewController.swift

```diff
@@ -21,15 +21,35 @@ import MaterialComponents.MaterialIcons_ic_color_lens
 import MaterialComponents.MaterialIcons_ic_help_outline
 import MaterialComponents.MaterialLibraryInfo
 
+class MDCMenuItem {
```

Comment:
where are we taking advantage of the `isButton` variable?

Also this seems more fitting for a struct than a class. Because we don't have a lot of items then passing it by value seems like a more reasonable approach.

Reply:
Good catch, it was left over from an earlier draft where I marked button-like menu items as buttons.  I've removed that in favor of hints.

Reply:
could we make this a struct?